src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer starts working");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Bulldozer stops working");
+    }
+}
```

Comment:
you should have empty line at the end of the file. Please fix here and in all other files